server/routes/index.ts

```diff
@@ -63,5 +66,44 @@ export default function routes(services: Services): Router {
   // NOMIS data dumps used for updating constants in this repository
   router.use('/download-report-config', makeDownloadConfigRouter())
 
+  // Digital Prison Reporting
+  get(
+    '/incident-reporting/summary',
+    ReportListUtils.createReportListRequestHandler({
+      title: 'Incident reports',
+      definitionName: 'incident-report',
+      variantName: 'summary',
+      apiUrl: config.apis.hmppsIncidentReportingApi.url,
+      apiTimeout: config.apis.hmppsIncidentReportingApi.timeout.deadline,
+      layoutTemplate: 'partials/reportsLayout.njk',
+      tokenProvider: defaultTokenProvider,
+    }),
+  )
+
+  get(
+    '/incident-reporting/count',
+    ReportListUtils.createReportListRequestHandler({
+      title: 'Incident Count',
+      definitionName: 'incident-count',
+      variantName: 'by-location-per-week',
+      apiUrl: config.apis.hmppsIncidentReportingApi.url,
+      apiTimeout: config.apis.hmppsIncidentReportingApi.timeout.deadline,
+      layoutTemplate: 'partials/reportsLayout.njk',
+      tokenProvider: defaultTokenProvider,
+    }),
+  )
+
+  get(
+    '/incident-reporting/prisoners',
+    ReportListUtils.createReportListRequestHandler({
+      title: 'Prisoners with incidents',
+      definitionName: 'incident-with-people',
+      variantName: 'by-prisoner',
+      apiUrl: config.apis.hmppsIncidentReportingApi.url,
+      apiTimeout: config.apis.hmppsIncidentReportingApi.timeout.deadline,
+      layoutTemplate: 'partials/reportsLayout.njk',
+      tokenProvider: defaultTokenProvider,
```

Comment:
is it the token that determines what a user can/cannot see? ie. we are deferring _all_ permissions checks to IRS api and/or DPR?

Reply:
Yes at the moment its the API/DPR that decides but I'd prefer this to change in the longer term, at the moment I've made it so that anyone with the read only role can access, but this will also be restricted by caseload etc.